programming/sublime-text/pspec.xml

```diff
@@ -20,17 +22,39 @@
     </Source>
 
     <Package>
-        <Name>sublime-text-3</Name>
+        <Name>sublime-text</Name>
+        <Summary xml:lang="en">Sublime Text is a sophisticated text editor for code, markup and prose.</Summary>
+        <Description xml:lang="en"> Sublime Text is a sophisticated text editor for code, markup and prose.</Description>
+        <PartOf>programming</PartOf>
         <Files>
```

Comment:
After some investigating, you can add a "Conflicts" section to directly replace the old sublime-text-3 on install

```
        <Conflicts>
            <Package>sublime-text-3</Package>
        </Conflicts>
```
Then there are changes needed to the Solus Software Center and the Help Center docs to incorporate the name change:

https://github.com/getsolus/solus-sc/blob/cbab7e083df75977bbaa2f01a42f89ae4a3b1477/solus_sc/thirdparty.py#L110

https://github.com/getsolus/help-center-docs/blob/master/software/third-party/en.md#sublime-text-3

Reply:
I will update this along with other comments by end of the week.